### ISSUE TITLE
Retina Macbook 12 inch 2016

### ISSUE THREAD

**Issue by MrBokeh** [Floris497/mac-pixel-clock-patch-V2#23]
Which script and version should I run for the Retina Macbook 12 inch 2016?
It has an Intel 515 chip in it.

**Comment by Floris497** [comment]
That is a good question.. what are you trying to acomplish?
Your computer supports 4k@60Hz out of the box..

**Comment by Floris497** [comment]
That is not true.. you'll need a [usb-c type DisplayPort] -> [DisplayPort] cable.. then you would need the IOKit patch.. i wonder if the core M's integrated graphics are powerful enough to take 4k@60Hz.. but i would be amazed if it could not deliver the resolution/frame rate apart from the fact that you are most probably going to see some lagging when doing graphics intensive tasks.

http://www.displayport.org/what-is-displayport-over-usb-c/
https://store.google.com/product/usb_type_c_to_displayport_cable

**Comment by shog86** [comment]
So can I make it work by using the official Apple adapter to connect New Macbook with my monitor via USB-c?

**Comment by Floris497** [comment]
It is useful for me to know what you want it to do.. Then i can tell you what you need and if it is possible. Or if there is a chance to make this possible.

**Comment by shog86** [comment]
I got a new Macbook with USB-C port, and I bought a third-party multimedia adapter to connect my Macbook to my 2560*1080 monitor via HDMI cable. But it didn't work, there's no signal transmitted to my monitor. And I can't use the adapter to connect my Macbook to my projector, eithter. So I wonder if there's something wrong with the adapter...and whether this problem will be solved if I buy an original Apple multimedia adapter.

**Comment by Floris497** [comment]
Yes.. But you can also go for the google one.. Although I don't think the Google one has charging capabilities. For the rather odd resolution you might need the IOKit patch.

**Comment by shog86** [comment]
but i encountered syntax error in line 5 or something...SIP is off...

**Comment by Floris497** [comment]
Don't hesitate to give full information. I cannot see what error might occur on line 5..

**Comment by shog86** [comment]
Here's the result in Terminal:

localhost:Downloads shog$ chmod +x IOKit-patcher.command

localhost:Downloads shog$ ~/Downloads/IOKit-patcher.command

/Users/shog/Downloads/IOKit-patcher.command: line 5: syntax error near
unexpected token `newline'

/Users/shog/Downloads/IOKit-patcher.command: line 5: `<!DOCTYPE html>'

localhost:Downloads shog$

On Fri, Jun 3, 2016 at 1:14 PM, Floris Fredrikze <notifications@github.com>
wrote:

> Don't hesitate to give full information. I cannot see what error might
> occur on line 5..
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/Floris497/mac-pixel-clock-patch-V2/issues/23#issuecomment-223492419>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ASzKd4Vd3REyue2Iv5puKI2TboDs4h0zks5qH7gwgaJpZM4IpRVg>
> .
>



-- 
葛仲君
Cell：15001246246

**Comment by Floris497** [comment]
Ah.. Common mistake.. You downloaded the weppage amd not the actual file. Download the zip. That works all the time :)

**Comment by shog86** [comment]
Thank you！~I'll have a try~

On Sun, Jun 5, 2016 at 2:47 PM, Floris Fredrikze <notifications@github.com>
wrote:

> Ah.. Common mistake.. You downloaded the weppage amd not the actual file.
> Download the zip. That works all the time :)
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/Floris497/mac-pixel-clock-patch-V2/issues/23#issuecomment-223796453>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ASzKd204wM5mBgj9tr4nEFxv0vQ7xkQzks5qInEXgaJpZM4IpRVg>
> .
>



-- 
葛仲君
Cell：15001246246

**Comment by Floris497** [comment]
Any news on this?

**Comment by heinmuek** [comment]
Bought a Hisense 40K3300UW, 40" 4K TV as I found 4K on the available 27"-28" monitors too much of an eye strain. Now a I need a deeper desk ;)
I have tried this patch a number of times now but whenever I restart my 2016 12' Macbook I still see "3840 x 2160 @ 30 Hz" in the System Report. Then when I open Monitors in System Preferences, I see it set to 30Hz BUT when I click on that menu I see a 60 HZ (NTSC) option. Unfortunately clicking on that option results in the monitor going black & even opening the MacBook & tapping any keys or the mouse does not bring either screen back to life. The caps lock key responds but only a hard shut down will get me out of that situation. Running 10.11.6.
Thanks
